package.json

```diff
@@ -21,7 +21,7 @@
     "dotenv": "^6.1.0",
     "gists": "^2.0.0",
     "node-cron": "^2.0.3",
-    "node-fetch": "^2.2.0",
+    "node-fetch": "^2.6.6",
```

Comment:
This change was unnecessary. the old version matches this version as well

Reply:
I did edit this